File: ContainerWithMostWater/brute_force.py
class Solution:
    def maxArea(self, height: List[int]) -> int:
        most = float('-inf')
        for i in range(len(height)):
            for j in range(i + 1, len(height)):
                curr = (j - i) * min([height[i], height[j]])
                most = max(most, curr)
        return most

File: ContainerWithMostWater/brute_force.java
public class Solution {
    public int maxArea(int[] height) {
        int most = Integer.MIN_VALUE;
        for (int i = 0; i < height.length; i++) {
            for (int j = i + 1; j < height.length; j++) {
                int curr = (j - i) * Math.min(height[i], height[j]);
                most = Math.max(most, curr);
            }
        }
        return most;
    }
}
